config.go

```diff
@@ -649,6 +651,8 @@ type moduleConfig struct {
 	environKeys map[string]int
 	// fsConfig is the file system configuration for ABI like WASI.
 	fsConfig FSConfig
+	// sockConfig is the network listener configuration for ABI like WASI.
+	sockConfig *internalsock.Config
```

Comment:
I see this is now sock instead fo net :)

Reply:
yeah conventionally sock won, both in syscall layer and in most of go's type names